lib/src/hexabase.dart

```diff
@@ -81,6 +83,20 @@ class Hexabase {
     return token != null;
   }
 
+  Future<HexabaseWorkspace> selectWorkspace(String workspaceId) async {
```

Comment:
Please change method name "selectWorkspace" to "current".
I'd like to use the same name as the JS SDK.

Reply:
@kondokentaro 
I have updated it, please check it for me ! 
Thank you!

Reply:
@lb-cuonghx 
LGTM!
Thank you very much!